Tasks/Linnik/Task2/task.py

```diff
@@ -0,0 +1,120 @@
+def to_next_hour(hhmm: list, to_next='we_need'):
+    """
+    Данная функция создана для уменьшения объёма словаря и перехода к следующему через уже знакомые числа.
+    hhmm: list,
+    next: str (like a switch),
+    return list[0]
+    """
+    if to_next == 'we_need':
+        if hhmm[0]+1 > 12:
+            hhmm[0] = hhmm[0]+1 - 12
+        else:
+            hhmm[0] = hhmm[0] + 1
+        return hhmm[0]
+    else:
+        if hh_mm[0] > 12:
+            hh_mm[0] = hh_mm[0] - 12
+        return hhmm[0]
+
+
+numbers = {
+    1: ['час', "первого"],
+    2: ["два", "второго", "двух"],
+    3: ["три", "третьего", "трёх"],
+    4: ["четыре", "четвёртого", "четырёх"],
+    5: ["пять", "пятого"],
+    6: ["шесть", "шестого"],
+    7: ["семь", "седьмого"],
+    8: ["восемь", "восьмого"],
+    9: ["девять", "девятого"],
+    10: ["десять", "десятого"],
+    11: ["одиннадцать", "одиннадцатого"],
+    12: ["двенадцать", "двенадцатого"],
+    13: ["тринадцать"],
+    14: ["четырнадцать"],
+    15: ["пятнадцать"],
+    16: ["шестнадцать"],
+    17: ["семнадцать"],
+    18: ["восемнадцать"],
+    19: ["девятнадцать"],
+    20: ["двадцать"],
+}
+
+print("Введите время в виде чч:мм")
+user_time = f"{input('Который час? ')}:{input('Сколько минут? ')}"
+
+hh_mm = []
+separated_time = user_time.split(':')
+hh_mm.append(int(separated_time[0]))
+hh_mm.append(int(separated_time[1]))
+del separated_time
```

Comment:
one single variable is a very small thing, usually we can leave it to be; manual deletion could be an option in case of a really huge object stored and here it would only consume time of interpreter and time of the code reader (to understand why we should really get rid of the var)